tests/19-enums-tuple-variant-incomplete.stderr

```diff
@@ -1,15 +1,18 @@
 error[E0308]: mismatched types
- --> $DIR/19-enums-tuple-variant-incomplete.rs:4:9
+ --> tests/19-enums-tuple-variant-incomplete.rs:4:9
   |
 4 | #[def = "A"]
   |         ^^^
   |         |
-  |         expected enum `Foo`, found fn item
+  |         expected `Foo`, found enum constructor
   |         expected `Foo` because of return type
-  |         help: use parentheses to instantiate this tuple variant: `"A"(_)`
 5 | enum Foo {
 6 |     A(usize),
-  |     -------- fn(usize) -> Foo {Foo::A} defined here
+  |     - `A` defines an enum variant constructor here, which should be called
   |
-  = note: expected enum `Foo`
-          found fn item `fn(usize) -> Foo {Foo::A}`
+  = note:          expected enum `Foo`
+          found enum constructor `fn(usize) -> Foo {Foo::A}`
+help: use parentheses to construct this tuple variant
+  |
+4 | #[def = "A"(/* usize */)]
+  |            +++++++++++++
```

Comment:
Shouldn't this be `"A(/* usize */)"`?

Reply:
It certainly should, yes. But I didn't find a way to reduce the span manually or get a sub-span of `syn::LitStr` without quotes by parsing (after parsing the span is the same). Probably relates to https://github.com/dtolnay/syn/issues/1219.